Eatery/Eat Now/Detail View Controller/EateriesCollectionViewGridLayout.swift

```diff
@@ -22,7 +22,7 @@ class EateriesCollectionViewGridLayout: UICollectionViewFlowLayout {
         itemSize = CGSize(width: cellWidth, height: cellWidth * 0.8)
         minimumLineSpacing = kCollectionViewGutterWidth
         minimumInteritemSpacing = kCollectionViewGutterWidth / 2
-        sectionInset = UIEdgeInsets(top: 2, left: kCollectionViewGutterWidth, bottom: 20, right: kCollectionViewGutterWidth)
+        sectionInset = UIEdgeInsets(top: 2, left: kCollectionViewGutterWidth, bottom: 0, right: kCollectionViewGutterWidth)
```

Comment:
Let's keep a slight inset here because otherwise the bottom of the collectionview has no standard size padding. Generally 8 or 12 points I think, gotta read me some Apple HIG again ;)